src/main/java/com/github/libgraviton/messaging/config/ContextProperties.java

```diff
@@ -0,0 +1,30 @@
+package com.github.libgraviton.messaging.config;
+
+import java.util.Properties;
+
+
+/**
+ * This class allows you to access a set of properties in a given config. For example if you define config "config",
+ * the call of getProperty("some.property") will return the value of config.some.property.
+ */
+public class ContextProperties extends Properties {
+
+    private String context;
+
+    /**
+     * Constructor.
+     *
+     * @param properties The properties instance holding all properties.
+     * @param context The config to be used.
+     */
+    public ContextProperties(Properties properties, String context) {
+        super(properties);
+        this.context = context + '.';
```

Comment:
The idea is that you don't need to suffix your context by `.`, which would most likely always be the case as `.` is the default delimiter for property names (at least in our libs). How about a `setContextDelimiter()`?